tools/authors/add.php

```diff
@@ -346,7 +346,7 @@ function echo_date_fields($bd) {
 <tr><td><?php echo _('Year'); ?>:</td><td>
 <input type="radio" name="<?php echo $bd; ?>yearRadio" value="0"<?php echo (_var($bd, 'yearRadio')=='0'?' CHECKED':''); ?>><?php echo _('Unknown'); ?>
 <br><input type="radio" name="<?php echo $bd; ?>yearRadio" value="1"<?php echo (_var($bd, 'yearRadio')=='1'?' CHECKED':''); ?> onClick="this.form.<?php echo $bd; ?>year.focus();" ><?php echo _('As entered'); ?>:
-<input type="text" name="<?php echo $bd; ?>year" size="4" maxlength="4"<?php echo (_var($bd, 'yearRadio')=='1'?' VALUE="'.abs(_var($bd, 'year')).'"':''); ?> onFocus="this.form.<?php echo $bd; ?>yearRadio[1].checked=true;">
+<input type="number" name="<?php echo $bd; ?>year" style='width: 4em;'<?php echo (_var($bd, 'yearRadio')=='1'?' VALUE="'.abs(_var($bd, 'year')).'"':''); ?> onFocus="this.form.<?php echo $bd; ?>yearRadio[1].checked=true;">
```

Comment:
Can't access this page, but it looks fine, maybe a `required` and `min` would not hurt.

Reply:
Missed this one in my first round of CR updates. Got it now.